addons/website/views/snippets/snippets.xml

```diff
@@ -833,6 +833,13 @@
             data-css-property="background-color"
             data-color-prefix="bg-"
             data-name="header_color"/>
+        <we-colorpicker string="Text Color"
+            class="o_we_sublevel_1"
+            id="option_header_transparent_text_color"
+            data-select-style="true"
+            data-css-property="color"
+            data-name="header_text_color"
+            data-color-prefix="text-"/>
```

Comment:
Same issue as with background color (infinite loader when we change opacity)

Reply:
FIXED

Reply:
GOOD TO KNOW (:smile:)